.scripts/backend/install.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+source .config/build
+
+cd $BE_SOURCE
+
+yarn install --frozen-lockfile
```

Comment:
This is the wrong install script - use `npm ci` instead.

Reply:
Obsolete now as those scripts were removed